Source/WebKit/UIProcess/API/Cocoa/WKSnapshotConfigurationPrivate.h

```diff
@@ -35,6 +35,10 @@ NS_ASSUME_NONNULL_BEGIN
 
 @property (nonatomic, setter=_setUsesTransparentBackground:) BOOL _usesTransparentBackground WK_API_AVAILABLE(macos(14.2), ios(17.2));
 
+#if !TARGET_OS_IPHONE
```

Comment:
I'm not sure I understand why this would be mac specific

Reply:
Only because we currently only need this for macOS, and iOS has a different code path (so it's not trivial to make the change and test) . We should consider adding it to iOS later.
Thanks for the review!